### DISCUSSION TITLE
How do I specify that my CustomNode does not take any content.

### DISCUSSION THREAD

**Discussion by aelkheir** [ueberdosis/tiptap#2633] [General Questions]
I am completely lost after spending 2 days on this. I made a custom node `CustomNode`, and want to use it inline like this:
`<p>`
`some text`
`<custom-node data='...'><custom-node>`
`another text`
`<p>`

However, the problem is the custom node is eating any text that comes after it. I think this is because I'm setting `content: 'inline'`.
Is there away to tell the schema that this node doesn't have any children. almost like a self-closing tag.

**Comment by cadars** [comment]
If your code is anything like the example you posted, you could first close your tags :)

If not, have you tried `content: 'inline*'`? https://tiptap.dev/api/schema#how-a-schema-looks-like

**Comment by philippkuehn** [comment]
It’s really hard to help without some example code!
